src/main/java/de/dhbw/mh/slang/craft/parser/CraftedSlangParser.java

```diff
@@ -77,318 +84,339 @@ private static final class Selector {
 		private static final Set<Token.Type> LITERAL1       = setOf( TRUE, FALSE );
 		private static final Set<Token.Type> LITERAL        = setOf( LITERAL1, NUMERIC_LITERAL );
 	}
-	
-	
-	
+
+
+
 	/*===========================================================
 	 * conditionalExpression
 	 *===========================================================*/
-	
+
 	@Override
 	public AstNode conditionalExpression( ){
 		// TODO Auto-generated method stub
 		return super.conditionalExpression( );
 	}
-	
+
 	/*===========================================================
 	 * conditionalOrExpression
 	 *===========================================================*/
-	
+
 	@Override
 	public AstNode conditionalOrExpression( ){
-		// TODO Auto-generated method stub
-		return super.conditionalOrExpression( );
+		switch(LEXER.lookahead().TYPE) {
+			case PLUS:
+			case MINUS:
+			case LPAREN:
+			case IDENTIFIER:
+			case NUMERIC_LITERAL:
+				return this.disjunction(conditionalAndExpression());
+			default:
+				Set<Token.Type> acceptedTypes = setOf(PLUS, MINUS, LPAREN, IDENTIFIER, NUMERIC_LITERAL);
+				throw this.parsingException(acceptedTypes);
+		}
 	}
-	
+
 	@Override
 	AstNode disjunction( AstNode previous ){
-		// TODO Auto-generated method stub
-		return super.disjunction( previous );
+		switch(LEXER.lookahead().TYPE) {
+			case LOR:
+				return this.disjunction1(previous);
+			case EOF:
+			case RPAREN:
+				return this.disjunction2(previous);
+			default:
+				throw this.parsingException(Selector.DISJUNCTION);
+		}
 	}
-	
+
 	@Override
 	AstNode disjunction1( AstNode previous ){
-		// TODO Auto-generated method stub
-		return super.disjunction1( previous );
+		if(LEXER.lookahead().TYPE != LOR) {
+			throw this.parsingException(LOR);
+		} else {
+            LEXER.advance();
+        }
```

Comment:
This is exactly what `match(Token.Type.LOR)` does. It's a little clearer that way.